docs/docs/guides/evaluation/guardrails_and_monitors.md

```diff
@@ -145,37 +145,134 @@ When applying scorers:
+   - **Calls to monitor**:
+        - **Operations**: Choose one or more `@weave.op`s to monitor. 
+        - **Filter** *(optional)*: Narrow down which op columns are eligible for monitoring (e.g., `max_tokens` or `top_p`)
+        - **Sampling rate**: The percentage of calls to be scored, between 0% and 100% (e.g., 10%)
+            :::tip Understanding `sampling_rate`
+            The `sampling_rate` controls what fraction of matching calls will be scored. It is a float between `0.0` and `1.0`:
+            | Value | Meaning                              |
+            | ----- | ------------------------------------ |
+            | `1.0` | Score all matching calls         |
+            | `0.5` | Score \~50% of calls (random)        |
+            | `0.1` | Score \~10% of calls                 |
+            | `0.0` | Score no calls (useful for dry-runs) |
+            :::
+   - **LLM-as-a-Judge configuration**: 
+        - **Scorer name**: Valid scorer names must start with a letter or number and can only contain letters, numbers, hyphens, and underscores.
+        - **Judge model**: Select the model that will score your ops and configure the following settings:
+            - **Configuration name**
+            - **System prompt**
+            - **Response format**
+        - **Scoring prompt**: The prompt used by the LLM-as-a-judge to score your ops. The prompt should specify what the `{output}` and `{input}` are.
+4. Click **Create Monitor**. Weave will automatically begin monitoring and scoring calls that match the specified criteria. You can view monitor details in the **Monitors** tab.
+
+### Example: Create a truthfulness monitor
+
+In the following example, you'll create:
+- The `weave.op` to be monitored, `generate_statement`. This function outputs statements that either returns the input `ground_truth` statement (e.g. `"The Earth revolves around the Sun."`), or generates a statement that is incorrect based on the `ground_truth` (e.g. `"The Earth revolves around Saturn."`)
+- A monitor, `truthfulness-monitor`, to evaluate the truthfulness of the generated statements.
+
+1. Define `generate_statement`:
+   ```python
+    import weave
+    import random
+    import openai
+
+    # Replace my-team/my-weave-project with your Weave team and project name 
+    weave.init("my-team/my-weave-project")
+
+    client = openai.OpenAI() 
+
+    @weave.op()
+    def generate_statement(ground_truth: str) -> str:
+        if random.random() < 0.5:
+            response = openai.ChatCompletion.create(
+                model="gpt-4.1",
+                messages=[
+                    {
+                        "role": "user",
+                        "content": f"Generate a statement that is incorrect based on this fact: {ground_truth}"
+                    }
+                ]
+            )
+            return response.choices[0].message["content"]
+        else:
+            return ground_truth
+   ```
+2. In the Weave UI, navigate to **Monitors**.
```

Comment:
2. should be "log a trace".

The `generate_statement` op will not appear in the Op dropdown unless it was logged at least once.